src/Tasks/Microsoft.NET.Build.Tasks/FrameworkPackages/FrameworkPackages.net6.0.cs

```diff
@@ -18,7 +18,7 @@ internal static class NETCoreApp60
             { "System.Formats.Asn1", "6.0.0" },
             { "System.Net.Http.Json", "6.0.0" },
             { "System.Reflection.Metadata", "6.0.0" },
-            { "System.Runtime.CompilerServices.Unsafe", "6.0.0" },
+            { "System.Runtime.CompilerServices.Unsafe", "6.1.2" },
```

Comment:
We can't change past versions for this, of course, because it was still shipping major versions/API.